src/components/DragDropContainer/DragDropContainer.tsx

```diff
@@ -56,7 +62,9 @@ export const DragDropContainer = ({
         {(provided: DroppableProvided) =>
           items.length > 0 ? (
             <ol
-              className={styles['drag-drop__container-inner']}
+              className={`${styles['drag-drop__container-inner']} ${
+                container.columnClassName && container.columnClassName
+              }`}
```

Comment:
nit: think we need to use `clsx` here and possibly extract the logic out into a variable for readability

Reply:
Was on the fence about that. But if that's what you all want, I can do that

Reply:
Yea otherwise, we get "undefined" as part of class name https://github.com/chanzuckerberg/edu-design-system/blob/264655eaef9ff57f70451f5ad02057080d34ff53/src/components/DragDrop/__snapshots__/DragDrop.test.tsx.snap#L32

Reply:
Just pushed that up. Thanks!